pkg/agents/team2/config.go

```diff
@@ -77,58 +73,111 @@ func InitTable(numStates int, numActions int) [][]float64 {
 }
 
 func New(baseAgent *infra.Base) (infra.Agent, error) {
-	stateSpace := InitStateSpace(3, 3, 3)
-	actionSpace := InitActionSpace()
-	policies := InitPolicies(27, 3)
-	rTable := InitTable(27, 3)
-	qTable := InitTable(27, 3)
+	actionDim := 6
+	daysAtCriticalDim := baseAgent.HealthInfo().MaxDayCritical + 1
+	stateSpace := InitStateSpace(10, 10, daysAtCriticalDim, 11)
+	actionSpace := InitActionSpace(actionDim)
+	policies := InitPolicies(10*10*daysAtCriticalDim*11, actionDim)
```

Comment:
```suggestion
	policies := InitPolicies(1100*daysAtCriticalDim, actionDim)
```
A suggestion. Similar in other variables